HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -315,14 +319,59 @@ wait_server(){
 	echo "server is ready!"
 }
 
+edit_model(){
+    cp -r $1 $TMPDIR/local_model_repo/
+    IFS='/' read -ra ADDR <<< "$1"
+    CONFIG=$TMPDIR/local_model_repo/${ADDR[-1]}/config.pbtxt
+
+    echo "" >> "$CONFIG"
+    echo "instance_group [" >> "$CONFIG"
+    echo "  {" >> "$CONFIG"
+    echo "    count: "$2 >> "$CONFIG"
+    echo "    kind: KIND_CPU" >> "$CONFIG"
+    echo "  }" >> "$CONFIG"
+    echo "]" >> "$CONFIG"
+    echo "" >> "$CONFIG"
+
+    if grep -Fq "onnx" "$CONFIG"
```

Comment:
to try to maintain a consistent style in this script, a few things to fix:
* indent with tabs (currently a mix of tabs and spaces in newly added lines, which is the worst of all possibilities)
* format if statements as:
  ```bash
  if condition; then
  	action
  fi
  ```

Reply:
Fixed.  Sorry, didn't notice that my emacs was doing space instead of tabs when I was pressing "tab"